clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -477,6 +489,8 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
     kafkaOpenSSLEnabled = serverProperties.getBoolean(SERVER_ENABLE_KAFKA_OPENSSL, false);
     helixHybridStoreQuotaEnabled = serverProperties.getBoolean(HELIX_HYBRID_STORE_QUOTA_ENABLED, false);
     ssdHealthCheckShutdownTimeMs = serverProperties.getLong(SERVER_SHUTDOWN_DISK_UNHEALTHY_TIME_MS, 200000);
+    sslHandshakeThreadPoolSize = serverProperties.getInt(SERVER_SSL_HANDSHAKE_THREAD_POOL_SIZE, 100);
+    sslHandshakeQueueCapacity = serverProperties.getInt(SERVER_SSL_HANDSHAKE_QUEUE_CAPACITY, 10000);
```

Comment:
What happens if the queue capacity exceeds?

10k might be small in some cases, I prefer the prev version of using MAX_INT and we can monitor the metrics to see what we should set.

Reply:
I think you just get rejected with an execution exception if queue is full and new task is asked to be executed. Will revert to int max then.